M2/Macaulay2/m2/last.m2

```diff
@@ -63,6 +63,7 @@ addStartFunction( () -> (
 	       	    hd := "with packages: ";
 	       	    stderr << hd << wrap(printWidth-#hd, concatenate between_", " sort apply(relevant,toString)) << endl;
 		    );
+	       stderr << "Try 'help', 'copyright', or 'cite' to get started." << endl;
```

Comment:
Perhaps be more verbose:
```
stderr << "Useful commands:  see 'copyright', how to 'cite', and get initial `help`." << endl;
```
We probably also want to put a tiny bit more info in the "initial help" provided via `help`. E.g., `loadedPackages`, `needsPackage`, `needs` (or `load`)...
Even more things like `errorDepth` (perhaps in `SeeAlso`?) could be helpful for a beginner.

Reply:
I think those things should go in `"getting started"` or `"a first Macaulay2 session"`.

The "Useful commands:" seems strange ... I get that `copyright` is important, but how is it useful?

Reply:
It is useful to get info on the copyright 😁